stream/cards.md

```diff
@@ -82,6 +105,10 @@ For security purposes, the Fidel API test environment doesn't accept live card n
 
 When linking a Fidel API test card, type in `$0.67` for verification.
 
+For 3D Secure, you may use one of these test card numbers:
+1. No verification required - `5186002600000005`
```

Comment:
Are these test cards that can be used with `pk_test` keys?

Reply:
Only with `live` keys, I think 🤔

Reply:
`pk_live`, I'm afraid, is due to a "test" logic they have on the endpoint, it needs to be live

Reply:
Are we allowed to make them public? I would think that we are not allowed as these are test cards coming from MC made available only for our use.